kernel/panic.c

```diff
@@ -212,7 +212,7 @@ void panic(const char *fmt, ...)
 	if (len && buf[len - 1] == '\n')
 		buf[len - 1] = '\0';
 
-	pr_emerg("Kernel panic - not syncing: %s\n", buf);
+	pr_emerg("Kernel bruh moment - not syncing: %s\n", buf);
```

Comment:
should be changed to `Kernel vibe check`

Reply:
take a look